csrankings-f.csv

```diff
@@ -504,6 +504,7 @@ Franz Puntigam,TU Wien,http://www.complang.tuwien.ac.at/franz,NOSCHOLARPAGE
 Franz W. Leberl,Graz University of Technology,http://www.leberl.info,-s4b1XAAAAAJ
 Franz Wotawa,Graz University of Technology,http://www.ist.tugraz.at/staff/wotawa,Z27gH2wAAAAJ
 Franziska Boehm,Karlsruhe Institute of Technology,https://www.zar.kit.edu/492.php?ID=237,NOSCHOLARPAGE
+Franziska Matthäus,Goethe University Frankfurt,https://www.fias.science/en/fellows/detail/matthaeus-franziska/,9F2kmU4AAAAJ

```

Comment:
Not in CS. Please explain how they meet inclusion criteria.

Reply:
Hi! Franziska Matthäus just became faculty at the Institute of Computer Science in Frankfurt in Oct/Nov, so sources might not have been updated yet (https://www.goethe-university-frankfurt.de/106106237/Professors?locale=en)